protoc_plugin/lib/indenting_writer.dart

```diff
@@ -101,8 +104,23 @@ class IndentingWriter {
     }
   }
 
+  void addSuffix(String suffixName, String text) {
+    suffixes[suffixName] = text;
+  }
```

Comment:
Documenting this method would be good. It's a bit surprising that keys are only used for to sort the printed lines and they're not printed.

(I wonder if we could use a better key type here? The string contents are not useful and are used to determine the order of lines. What would be a better key type for this?)

Reply:
+1 - I'll add some dartdoc here.

The keys are really the names of the fields that we emit (the field names are the keys, and the contents of the entire declaration - name and all - are the text). But the key is really meant to be opaque to the method here - just something to keep the suffixes unique. A field declaration will be written and over-written several times when generating a file - we just depend on the usage pattern here that the name and the contents are the same each time.

Reply:
I changed the param name to `suffixKey` and indicated in the dartdoc that we don't emit that param, just use it for suffix uniqueness.